Feat: Adds kongctl to developer (#4047)
Kong/developer.konghq.com: app/kongctl/authentication.md

 Or copy this one-time code: KFVL-RXXJ

 And open your browser to https://cloud.konghq.com/device-activate

 (Code expires in 899 seconds)

 Waiting for user to Login...
```

After following the instructions in the browser and successfully authenticating,
you will see the message `User successfully authorized`.

This indicates that kongctl has negotiated with {{site.konnect_short_name}} and 
stored an access and refresh token pair for subsequent commands. 

You can verify authentication by running:

```bash
kongctl get me
```

You should see your {{site.konnect_short_name}} user information.

Now you can execute kongctl commands and you will be granted access based on the permissions
of the user account you logged in with.

{:.info}
> **Info:** The tokens obtained using the browser-based method will expire. When they do you can
> simply re-exeucte the `kongctl login` procedure to obtain new tokens.

If you want to invalidate the token received from the browser-based method, 
execute the `logout` command to clear stored credentials:

```bash
kongctl logout
```

## Configured access token

{{site.konnect_short_name}} access tokens come in two forms; Personal Access Tokens (PAT) or 

> System Access Tokens (sPAT). PATs grant access to APIs as your personal user account, while 

Reviewer: ```suggestion
[{{site.konnect_short_name}} access tokens](/konnect-api/#personal-access-tokens) come in two forms: Personal Access Tokens (PAT) or 
System Access Tokens (sPAT). PATs grant access to APIs as your personal user account, while 
```

sPATs grant access based on the permissions of system account, which may be more 
limited than a user account. 

> Use the {{site.konnect_short_name}} web console to create the token type of your choice, and 

Reviewer: ```suggestion
Use the {{site.konnect_short_name}} UI to create the token type of your choice, and 
```

retain the secret value for use with kongctl:
- Create a PAT in the [personal access token page](https://cloud.konghq.com/global/account/tokens)
- Create a sPAT in the [system accounts page](https://cloud.konghq.com/global/account/system-tokens)
  or with the [System Accounts API](/api/konnect/identity/#/operations/post-system-accounts-id-access-tokens)

### Configure authentication via flag

You can pass the token with each command using the `--pat` flag:

```bash
kongctl get apis --pat "kpat_your-token-here"
```

### Configure access token in environment variable

Store the token in an environment variable to avoid passing it with every command. For the `default` profile,
set the `KONGCTL_DEFAULT_KONNECT_PAT` environment variable:

{:.info}
> See the [environment variable configuration reference](/kongctl/config#environment-variables) 
> for full details on environment variables and the kongctl configuration system 

```bash

> export KONGCTL_DEFAULT_KONNECT_PAT="kpat_your-token-here"

Reviewer: ```suggestion
export KONGCTL_DEFAULT_KONNECT_PAT="YOUR PAT"
```
